Extensions/xkit_patches.js

```diff
@@ -742,7 +742,67 @@ XKit.extensions.xkit_patches = new Object({
 						avatar: post.blog.avatar[post.blog.avatar.length - 1].url,
 						tags: post.tags.join(","),
 					};
+				},
+			};
+
+			XKit.interface.async_form_key = async function() {
+				const request = await fetch('https://www.tumblr.com/settings/dashboard');
+				const meta_tag = (await request.text()).match(
+					/tumblr-form-key[^<]*content=["']?(\w+)["']?/
```

Comment:
allows tumblr-form-keycontent=asdf which is a little invalid looking

Reply:
@hobinjk valid under the HTML spec!

Reply:
i removed it in favor of a more spec-compliant regex that actually handles non-\w characters correctly.